src/helpers/training.jl

```diff
@@ -160,6 +161,7 @@ Compute the gradients of the objective function wrt parameters stored in `ts`.
 | `AutoReverseDiff(; compile)` | `ReverseDiff.jl` |
 | `AutoTracker`                | `Tracker.jl`     |
 | `AutoEnzyme`                 | `Enzyme.jl`      |
+| `AutoForwardDiff`            | `ForwardDiff.jl` |
```

Comment:
```suggestion
| `AutoForwardDiff`            |                  |
```

we already have it loaded so no extra packages are needed